examples/animation/animation_masks.rs

```diff
@@ -0,0 +1,365 @@
+//! Demonstrates how to use masks to limit the scope of animations.
+
+use bevy::{animation::AnimationTargetId, color::palettes::css::WHITE, prelude::*};
+
+// IDs of the mask groups we define for the running fox model.
+//
+// Each mask group defines a set of bones for which animations can be toggled on
+// and off.
+const MASK_GROUP_LEFT_FRONT_LEG: u32 = 0;
+const MASK_GROUP_RIGHT_FRONT_LEG: u32 = 1;
+const MASK_GROUP_LEFT_HIND_LEG: u32 = 2;
+const MASK_GROUP_RIGHT_HIND_LEG: u32 = 3;
+const MASK_GROUP_TAIL: u32 = 4;
+
+// The width in pixels of the small buttons that allow the user to toggle a mask
+// group on or off.
+const MASK_GROUP_SMALL_BUTTON_WIDTH: f32 = 150.0;
+
+// The ID of the animation in the glTF file that we're going to play.
+const FOX_RUN_ANIMATION: usize = 2;
+
+// The names of the bones that each mask group consists of. Each mask group is
+// defined as a (prefix, suffix) tuple. The mask group consists of a single
+// bone chain rooted at the prefix. For example, if the chain's prefix is
+// "A/B/C" and the suffix is "D/E", then the bones that will be included in the
+// mask group are "A/B/C", "A/B/C/D", and "A/B/C/D/E".
+//
+// The fact that our mask groups are single chains of bones isn't anything
+// specific to Bevy; it just so happens to be the case for the model we're
+// using. A mask group can consist of any set of animation targets, regardless
+// of whether they form a single chain.
+static MASK_GROUP_PATHS: [(&str, &str); 5] = [
```

Comment:
Is there a reason for this to be a `static` instead of a `const`? I don't think it'd make a difference performance-wise, but I could imagine someone learning Bevy might be confused as to why this constant is treated differently to the others.

Reply:
Changed to a const.